pkg/cmd/delete/variables/variables_test.go

```diff
@@ -29,7 +29,7 @@ func TestCreate(t *testing.T) {
 		_, err := cmd.ExecuteC()
 		require.NoError(t, err)
 
-		assert.Equal(t, "🚀 Variable 7a187044-4a00-4a4a-93ed-d230900421f3 was successfully deleted\n\n", stdout.String())
+		assert.Equal(t, "Variable 7a187044-4a00-4a4a-93ed-d230900421f3 was successfully deleted\n", stdout.String())
```

Comment:
Use the message variable to centralize maintenance in tests

Reply:
remove \n to leave the next line formatting in the output method and be centered leaving only the message